src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -46,7 +108,7 @@
 					<template v-if="!initialisedConversations">
 						<LoadingPlaceholder type="conversations" />
 					</template>
-					<Hint v-else-if="searchText && !conversationsList.length"
+					<Hint v-else-if="(searchText && !conversationsList.length) || (!conversationsList.length && isFiltered)"
```

Comment:
```suggestion
					<Hint v-else-if="(searchText || isFiltered) && !conversationsList.length"
```
Easier to read and understand, right? And it's big enough, we could move it to computed property